components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -290,6 +291,14 @@ private void buildOAuthServerConfiguration() {
         OMElement scopeValidatorsElem = oauthElem.getFirstChildWithName(
                 getQNameWithIdentityNS(ConfigElements.SCOPE_VALIDATORS));
 
+        //Get scopeValidationEnabledConfigValue
```

Comment:
Please use the proper commenting style in all the places.

Reply:
Fix https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1294